src/notebook/index.css

```diff
@@ -46,6 +43,16 @@
 }
 
 
+.jp-Notebook::after {
+  content: ' ';
+  width: 100%;
+  min-height: 100px;
```

Comment:
Ideally we'd have enough space so that the bottom could be scrolled up to the top of whatever area we had. `90vh` should give us just less than one viewport high, though that may be too much if the document is actually a small tab inside the browser window.